sim/visuals/microbit.ts

```diff
@@ -966,7 +976,7 @@ path.sim-board {
 
         private buildDom() {
             this.domHardwareVersion = 1;
-            this.element = <SVGSVGElement>svg.elt("svg")
+            this.element = <SVGSVGElement>svg.elt("svg");
```

Comment:
There are a few lines like this where we've added semis that distract from this PR's focus. Please can we remove to keep a simple diff?